react/features/overlay/middleware.js

```diff
@@ -1,11 +1,21 @@
 // @flow
 
+import { JitsiConferenceErrors } from '../base/lib-jitsi-meet';
 import { StateListenerRegistry } from '../base/redux';
 
 import { setFatalError } from './actions';
 
 declare var APP: Object;
 
+/**
+ * List of errors that are not fatal (or handled differently) so then the overlays won't kick in.
+ */
+const NON_FATAL_ERRORS = [
+    JitsiConferenceErrors.CONFERENCE_ACCESS_DENIED,
+    JitsiConferenceErrors.CONFERENCE_DESTROYED,
+    JitsiConferenceErrors.CONNECTION_ERROR
```

Comment:
I think this masks a legit authentication error: https://github.com/jitsi/lib-jitsi-meet/blob/6af8eee57d1ebdc0881c8c2875d4346e02d01549/modules/xmpp/ChatRoom.js#L1101